docs/running.pod

```diff
@@ -30,6 +30,12 @@ compiled code.
   --profile            write profile information as HTML file (MoarVM)
   --profile-filename   provide a different filename (also allows .json)
   --doc=[module]       Use Pod::To::[module] to render inline documentation.
+  --dump               dump the bytecode to stdout instead of executing (MoarVM)
```

Comment:
Hmm, not sure how useful this is, in that it doesn't work with Perl 6 input at all, but only a MoarVM bytecode file (assuming it just forwards to `moar --dump`)?

Reply:
`./perl6 --dump -e 'say "Hi"'` does seem to do stuff.

Reply:
Wow, it does. It appears to dump the bytecode of the compiler entrypoint program, though, not the script given to `-e`!

Reply:
It'll make sense to remove the argument altogether then. I'll see what I can do.

Reply:
(To add: it's not obvious how to wire the pieces together to actually use that dumper with the input given to the compiler.)